src/hooks/routing/types.ts

```diff
@@ -1,10 +1,19 @@
 export enum RouterPreference {
-  PRICE,
-  SKIP,
   API,
```

Comment:
I really like how you used string-equivalents for the new `enum QuoteType`. It eliminates an ambiguity of default values because `0` is falsey, so you can't check for `RouterPreference | undefined` with a boolean check.

Would you add explicit string values for `enum RouterPreference` too?